src/command.ts

```diff
@@ -14,5 +14,7 @@ export const onCommandReceived = async (command: Command, responseTopic: string)
 
   const commandStatus = await executeCommand(command);
 
+  console.log({ commandStatus, responseTopic, correlationData: command.id });
```

Comment:
Do we wanna add a prefix, to understand the context of the log? (e.g. onCommandRecieved - rest...)

Reply:
You already have context from the event received in the subscriber